example/test_problem/Hydro/GrackleTest/yt_script/plot_grackle_analytical_cooling_rate.py

```
@@ -0,0 +1,227 @@
+    cbar.set_ticklabels(tick_labels, fontsize=font_size)
+    cbar.ax.tick_params(direction='in', length=5, width=1.5)
+
+    # Set a thicker frame (spines)
+    for spine in ax.spines.values():
+        spine.set_linewidth(2.0)
+    for spine in cbar.ax.spines.values():
+        spine.set_linewidth(2.0)
+
+    # Add text annotations
+    ax.text(0.03, 0.97,
+            f'max = {net_rates.max():.3e} erg/(cm$^3 \cdot$ s)\nmin = {net_rates.min():.3e} erg/(cm$^3 \cdot$ s)',
```

Ruff (0.15.5)

[warning] Invalid escape sequence: `\c`

Add backslash to escape sequence

(W605)